parsec/core/cli/human_find.py

```diff
@@ -0,0 +1,37 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2016-2021 Scille SAS
+
+import click
+
+from parsec.utils import trio_run
+from parsec.core import logged_core_factory
+from parsec.core.cli.utils import cli_command_base_options, core_config_and_device_options
+from parsec.cli_utils import cli_exception_handler
+
+
+async def _human_find(config, device, query, page, per_page, omit_revoked, omit_non_human):
+    async with logged_core_factory(config, device) as core:
+        user_info_tab, nb = await core.find_humans(
+            query, page, per_page, omit_revoked, omit_non_human
+        )
+    for user in user_info_tab:
+        click.echo(f"{user.human_handle} - UserID: {user.user_id}")
+    if not nb:
+        click.echo("No human found!")
+
+
+@click.command(short_help="search user id from email address")
+@click.option("--query", required=True, type=str, help="Search through names and email addresses")
```

Comment:
Maybe query can be a`click.argument instead of being a `click.option` to be used directly (as it is the main entry for this command)